.circleci/config.yml

```diff
@@ -52,15 +52,26 @@ jobs:
             cibuildwheel --output-dir wheelhouse
           environment:
             CIBW_BUILD: << parameters.cibw-build >>
+
       - run:
-          name: Upload wheels
+          name: Install Anaconda Client & Upload Wheels
           command: |
-            if [[ -n ${TAG} ]]; then
-                export IS_PUSH="true"
-            fi
-            if [[ $TRIGGER_SOURCE == "scheduled_pipeline" ]]; then
-                export IS_SCHEDULE_DISPATCH="true"
-            fi
+            echo "Install Mambaforge"
+            MAMBA_URL="https://github.com/conda-forge/miniforge/releases/download/4.14.0-0/Mambaforge-4.14.0-0-Linux-aarch64.sh"
+            echo "Downloading $MAMBA_URL"
+            wget -q $MAMBA_URL -O minimamba.sh
+            chmod +x minimamba.sh
+
+            MAMBA_DIR="$HOME/miniconda3"
+            rm -rf $MAMBA_DIR
+            ./minimamba.sh -b -p $MAMBA_DIR
+
+            export PATH=$MAMBA_DIR/bin:$PATH
+
+            mamba install -y -c conda-forge anaconda-client
+            
+            anaconda -q -t ${TOKEN} upload --skip -u ${ANACONDA_ORG} ./wheelhouse/*.whl

```

Comment:
Just testing that the anaconda command can be found, will revert when its merge time.